### ISSUE TITLE
Parse Dashboard only showing a blank white page

### ISSUE THREAD

**Issue by benjaminchevoor** [parse-community/parse-dashboard#1596]
_Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Dashboard!_

- [✔] _You're running version >=1.0.23 of Parse Dashboard._

- [✔] _You're running version >=2.3.2 of Parse Server._

- [✔] _You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before._

I know this symptom has come up dozens of times, and I have gone through some dozen tickets and SO pages to try and resolve it. Nothing has worked for me so far.

#### Environment Setup
Docker

I am seeing this:
![image](https://user-images.githubusercontent.com/4171696/94587630-dc348200-0250-11eb-8e83-c956fbf78fb7.png)

#### Steps to reproduce
I am following the steps in this Medium article: 
https://medium.com/@yildizberkay/how-to-install-parse-server-with-docker-c5931bd84c17

Which can be summarized as:
```
docker run --name parse-mongo-db \
-e MONGO_INITDB_ROOT_USERNAME=admin \
-e MONGO_INITDB_ROOT_PASSWORD=admin \
-d mongo

docker run --name parse-server-1 \
 -e PARSE_SERVER_APPLICATION_ID=APP_ID_1 \
 -e PARSE_SERVER_MASTER_KEY=MASTER_KEY_1 \
 -e PARSE_SERVER_DATABASE_URI=mongodb://admin:admin@mongo/parse_server?authSource=admin \
 --link parse-mongo-db:mongo \
 -d -p 1337:1337 parseplatform/parse-server

docker run --name parse-dashboard-1 \
 -e PARSE_DASHBOARD_APP_ID=APP_ID_1 \
 -e PARSE_DASHBOARD_MASTER_KEY=MASTER_KEY_1 \
 -e PARSE_DASHBOARD_USER_ID=admin \
 -e PARSE_DASHBOARD_USER_PASSWORD=admin \
 -e PARSE_DASHBOARD_ALLOW_INSECURE_HTTP=true \
 -e PARSE_DASHBOARD_SERVER_URL=http://localhost:1337/parse \
 -d -p 4040:4040 parseplatform/parse-dashboard
```

#### Logs/Trace

From the chrome console:
```
VM115:1 POST http://localhost:1337/serverInfo net::ERR_CONNECTION_REFUSED
VM115:1 POST http://localhost:1337/schemas net::ERR_CONNECTION_REFUSED
VM115:1 POST http://localhost:1337/schemas net::ERR_CONNECTION_REFUSED
VM115:1 POST http://localhost:1337/schemas net::ERR_CONNECTION_REFUSED
VM115:1 POST http://localhost:1337/schemas net::ERR_CONNECTION_REFUSED
VM115:1 POST http://localhost:1337/schemas net::ERR_CONNECTION_REFUSED
```

**Comment by benjaminchevoor** [comment]
In writing up this original post I actually solved the issue. (Don't you love it when that happens, Mr. Rubber Duck?) But I wanted to post it anyway so I can share what I learned.

For me, the issue was in the environment variable `PARSE_DASHBOARD_SERVER_URL=http://localhost:1337/parse`.

Because I was accessing the server from another system on the network, I needed to fix this to be the IP address of the system hosting my docker container. Eg: `PARSE_DASHBOARD_SERVER_URL=http://192.168.2.39:1337/parse`.

And now I can see my dashboard!

I am good with marking this ticket as closed :)

**Comment by InTheClodus** [comment]
Thank you very much. Your article has helped me successfully. I have revised it for an hour before that. Thank you very much

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-alpha.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-alpha.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0)
